### PR TITLE
Increase the readability of perf event logs.

### PR DESCRIPTION
**Pull request by Creatone**
Signed-off-by: Paweł Szulik <pawel.szulik@intel.com>

### CODE REVIEW

**Comment by Creatone** [comment]
@bobbypage can you check this one ?

**Comment by bobbypage** [comment]
LGTM, thanks.

File: perf/uncore_libpfm.go [hunk 1]
```diff
@@ -151,8 +151,7 @@ func NewUncoreCollector(cgroupPath string, events PerfEvents, cpuToSocket map[in
 
 	err := collector.setup(events, systemDevicesPath)
 	if err != nil {
-		formatedError := fmt.Errorf("unable to setup uncore perf event collector: %v", err)
-		klog.V(5).Infof("Perf uncore metrics will not be available: %s", formatedError)
+		klog.Errorf("Perf uncore metrics will not be available: unable to setup uncore perf event collector: %s", err)
```
Comment (bobbypage) [inline perf/uncore_libpfm.go:154]: shouldn't we use %v instead of %s here for the error?

**Review by bobbypage — CHANGES_REQUESTED** [review]
small nit regarding the formatting of error, otherwise LGTM

File: perf/uncore_libpfm.go (hunk 1 above)
Comment (Creatone) [reply perf/uncore_libpfm.go]: Yes

**Review by bobbypage — APPROVED** [review]

**Pull request merged**